docs/flow/reference/config-blocks/_index.md

```diff
@@ -6,3 +6,13 @@ weight: 200
 ---
 
 # Configuration blocks
+
+Configuration blocks are optional top-level blocks which can be used to
+configure various parts of the Grafana Agent process. Each config block may
```

Comment:
More picky stuff: Use _can_ or _might_ rather than _may_. 

(from Google developer style guide, which the Docs Squad follows): 

"In general, reserve _may_ for official policy or legal considerations. To convey possibility, use _can_ or _might_ instead. To convey permission, use _can_ instead.